### ISSUE TITLE
competition status should be already translated in combo list

### ISSUE THREAD

**Issue by scorreia** [dmossakowski/additive-spotify-analyzer#38]
<img width="641" alt="combolist" src="https://user-images.githubusercontent.com/6770495/174454433-1eb71574-d1d8-4c90-94e9-05e138901530.png">

**Comment by dmossakowski** [comment]
Yes .. this is for only select few people to see so we can do this later
